frontend/letsdostuff.frontend/src/containers/Registration.js

```diff
@@ -15,9 +18,31 @@ export default function Registration() {
   const [confirmPassword, setConfirmPassword] = useState("");
   //const [dateOfBirth, setDateOfBirth] = useState("");
   const [bio, setBio] = useState("");
+  const notifyError = () => {
+    toast.error('⚠️ Error!', {
```

Comment:
Seems like it is better to move this common behaviour to a separate service, but I am not sure